.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--- Make sure to add a description of your changes in docs/user_guide/installation_and_version_guide/v<major>.<minor>.<patch>.rst -->
+<!--- Feel free to add your name to AUTHORS.md. -->
```

Comment:
Should we drop that file?

Reply:
I believe the idea of that file was to be able to also add contributors who did not commit code directly but did contribute in other ways to the project (is that correct, @romanlutz?). Although I like the sentiment, we have not maintained the document very carefully and the added value is fairly small in my opinion. So I would also be in favor to drop it. If we still want to thank folks who contributed in different ways we might be able to find a spot on the website to acknowledge them?

Reply:
FWIW I've been wanting to write a script that generates the summary of all PRs (and the related issues) including authors and gives credit to all reviewers/commenters so that we can include that in the version file. That would mean nobody has to manually write it anymore and AUTHORS.md is obsolete. I haven't done this yet, of course...

I have no problem getting rid of AUTHORS.md now, or alternatively when I get to this. It may be a while, though.

Reply:
I'm fine dropping as well.

Reply:
Done.